src/store/store.js

```diff
@@ -493,11 +494,36 @@ export default new Vuex.Store({
 			}
 
 			const contexts = state.contexts
-			const index = contexts.findIndex(t => t.id === id)
+			const index = contexts.findIndex(c => c.id === id)
 			contexts.splice(index, 1)
 			commit('setContexts', [...contexts])
 			return true
 		},
+		async removeContext({ dispatch, state, commit }, { context }) {
+			try {
+				await dispatch('unShareContext', { receivers: context.sharing })
```

Comment:
I would actually prefer to just send a delete request and the backend would not only delete the context but also any shares, pages, app menu entries

@blizzz Any objections? Currently the ContextService (called by ContextController::delete) just drops the context entry it seems

Reply:
it works, thanks @blizzz